llvm/unittests/DebugInfo/PDB/PDBVariantTest.cpp

```diff
@@ -0,0 +1,56 @@
+//===- llvm/unittest/DebugInfo/PDB/PDBVariantTest.cpp ---------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "gtest/gtest.h"
+
+#include "llvm/DebugInfo/PDB/PDBTypes.h"
+
+using namespace llvm;
+using namespace llvm::pdb;
+
+template <typename T>
```

Comment:
It seems clang-format would prefer to not have a line break here.

Reply:
Fixed